pkg/logs/processor/processor.go

```diff
@@ -217,3 +220,24 @@ func (p *Processor) GetHostname(msg *message.Message) string {
 	}
 	return hname
 }
+
+func validateAndFilterRules(rules []*config.ProcessingRule) []*config.ProcessingRule {
+	var validRules []*config.ProcessingRule
+
+	for _, rule := range rules {
+		// Rules that require regex
+		if rule.Type == config.ExcludeAtMatch ||
```

Comment:
Non-blocking, but for the sake of code clarity, it might be worth it to refactor this `if` statement and the inner `if` statement to something like:
```go
ruleRequiresRegex := rule.Type == config.ExcludeAtMatch || rule.Type == config.IncludeAtMatch || rule.Type == config.MaskSequences
ruleRegexIsNil := rule.Regex == nil
if ruleRequiresRegex && ruleRegexIsNil {
    ...
}
```

You could even move the booleans outside the `for` loop so you don't have to allocate a new variable every run.

Reply:
https://grugbrain.dev/#grug-on-expression-complexity